css/all-devices-style.css

```diff
@@ -0,0 +1,754 @@
+.deals_item h4 {
+  margin-bottom: 12px;
+}
+.deals_heading {
+  display: flex;
+  justify-content: space-between;
+  margin: 50px 0;
+  color: #ff4422;
+}
+.deals_heading h3 {
+  font-weight: bold;
+  font-size: 30px;
+}
+.img_deal1 {
+  background: url(../images/sale_1_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal2 {
+  background: url(../images/sale_2_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal3 {
+  background: url(../images/sale_3_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal4 {
+  background: url(../images/sale_4_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal5 {
+  background: url(../images/sale_5_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal6 {
+  background: url(../images/sale_6_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal7 {
+  background: url(../images/sale_7_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+.img_deal8 {
+  background: url(../images/sale_8_primary_image.jpg);
+  background-color: red;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 280px;
+  height: 200px;
+  margin-bottom: 12px;
+}
+/*Footer*/
+.footer-top {
+	background-color: #131313;
+	padding: 30px;
+	height: 160px;
+	margin-bottom: -13px;
+	padding-bottom: 0px;
+}
+
+.iwraper {
+	width: 100%;
+	text-align: center;
+}
+
+.footer-top i {
+	background-color: #333;
+	color: #fff;
+	padding: 15px;
+	margin: 0px 0.3%;
+	border-radius: 50%;
+	width: 45px;
+	height: 45px;
+	cursor: pointer;
+	border: 1px solid #333;
+}
+
+#wifi,
+#envelope {
+	padding-left: 13px;
+}
+
+.footer-top i:hover {
+	background-color: #131313;
+}
+
+.footer-text-wraper {
+	padding-top: 30px;
+	color: #aaa;
+	font-size: 22px;
+	width: 100%;
+	text-align: center;
+}
+
+.footer-text-wraper span {
+	margin: 0px 5px;
+	cursor: pointer;
+}
+
+.footer-text-wraper span:hover {
+	color: #fff;
+}
+
+.footer-bottom {
+	width: 100%;
```

Comment:
For the bottom footer section. We can just remove the `width: 100%` and it will work just fine, or we can use `border-box` for our `box-sizing` property, but it may break another things. So let's stick with just removing `width: 100%` and consider using `box-sizing: border-box` in upcoming projects.